coffee_maker_navigator_2/lib/main.dart

```diff
@@ -1,68 +1,28 @@
+import 'package:coffee_maker_navigator_2/app.dart';
 import 'package:coffee_maker_navigator_2/data/di/data_di_module.dart';
-import 'package:coffee_maker_navigator_2/di/dependency_injection.dart';
-import 'package:coffee_maker_navigator_2/di/di/container_manager.dart';
-import 'package:coffee_maker_navigator_2/di/di/containers.dart';
-import 'package:coffee_maker_navigator_2/di/di/injector.dart';
+import 'package:coffee_maker_navigator_2/di/di/dependency_container_manager.dart';
+import 'package:coffee_maker_navigator_2/di/di/dependency_containers/orders_dependency_container.dart';
 import 'package:coffee_maker_navigator_2/domain/di/domain_di_module.dart';
 import 'package:coffee_maker_navigator_2/ui/di/ui_di_module.dart';
-import 'package:coffee_maker_navigator_2/ui/router/view/app_router_delegate.dart';
-import 'package:coffee_maker_navigator_2/ui/router/view_model/router_view_model.dart';
-import 'package:demo_ui_components/demo_ui_components.dart';
 import 'package:flutter/material.dart';
-import 'package:provider/provider.dart';
+
+void _registerDependencyContainerFactories() {
+  final manager = DependencyContainerManager.instance;
+  manager.registerContainerFactory((_) => OrdersDependencyContainer());
```

Comment:
should be with type <OrdersDependencyContainer>, otherwise it will be dynamic and all containers will overwrite each other and might not be resolved.

Reply:
How can we force the Type in use?

Reply:
Indeed this caused problems

Reply:
assert(C != dynamic, 'The type parameter C must be explicitly provided.');

Reply:
done 0724bed386db2016631f18bba8d2b6859cf239d7